src/site-server.ts

```diff
@@ -99,6 +101,7 @@ export class SiteServer {
 			...this.details,
 			url: this.server.url,
 			port: this.server.options.port,
+			phpVersion: this.server.options.phpVersion || DEFAULT_PHP_VERSION,
```

Comment:
Nitpick: The  `phpVersion` can be a supported value or `undefined`.
```suggestion
			phpVersion: this.server.options.phpVersion ?? DEFAULT_PHP_VERSION,
```